unity_packer/debug_single_export_json/analytics.json

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
Are all of these just the usual export, but with just App and the product they care about?  It feels bad having to maintain an extra set of files for this.

Reply:
Yes the export jsons are like what you described. I used a script to generate those so should not be too much maintain work. I also upload that script if that's better.

Reply:
I think it is fine for now. In the future we can look into making it use the main one, but we do rarely touch these files in general.